post-payments.php

```diff
@@ -0,0 +1,197 @@
+<?php
+/**
+ * Plugin Name: Post Payments
+ * Description: Tracks story cost and payment due totals.
+ * Version: 1.0
+ * Author: Matt Johnson, Alley Interactive
+ *
+ * 
+ */
+
+class Post_Payments {
+
+	public $default_post_types = 'post';
+	public $settings_option = 'post-payments-settings';
+	public $meta_key = 'post_cost';
+	public $currency_symbol = '$';
+
+	public function __construct() {
+
+		add_action( 'init', array( $this, 'add_meta_boxes' ) );
```

Comment:
maybe not right now, but it'd be an improvement to iterate over the post types and hook on `"fm_post_{$slug}"`